pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/UpdateActivityBaseSettings.java

```diff
@@ -107,14 +107,14 @@ private void compareBasicSettings(Handle handle, Config definition, long activit
                     latestDto.isInstantiateUponRegistration(),
                     latestDto.getMaxInstancesPerUser(),
                     latestDto.getEditTimeoutSec(),
-                    latestDto.isOndemandTriggerAllowed(),
-                    latestDto.shouldExcludeFromDisplay(),
+                    latestDto.isOnDemandTriggerAllowed(),
+                    latestDto.isExcludeFromDisplay(),
                     latestDto.isUnauthenticatedAllowed(),
                     latestDto.isFollowup(),
-                    latestDto.shouldExcludeStatusIconFromDisplay(),
+                    latestDto.isExcludeStatusIconFromDisplay(),
                     latestDto.isHideExistingInstancesOnCreation(),
                     latestDto.isCreateOnParentCreation(),
-                    latestDto.canDeleteInstances(),
+                    latestDto.isCanDeleteInstances(),
```

Comment:
Methodically adding the `is` prefix makes for very awkwardly named properties here.